doc/cprover-manual/cbmc-tutorial.md

```diff
@@ -1,23 +1,5 @@
 [CPROVER Manual TOC](../../)
 
-## Bounded Model Checking with CBMC
```

Comment:
Where is this duplicated?

Reply:
https://github.com/diffblue/cbmc/blame/develop/doc/cprover-manual/introduction.md#L24